src/frontend/src/utils/iiConnection.ts

```diff
@@ -966,6 +969,60 @@ export class AuthenticatedConnection extends Connection {
     if ("Err" in res) throw new CanisterError(res.Err);
   };
 
+  fromJwt = async (
+    jwt: JWT,
+    salt: Salt,
+    maxTimeToLive?: bigint
+  ): Promise<AuthenticatedConnection> => {
+    const sessionIdentity = await ECDSAKeyIdentity.generate({
+      extractable: false,
+    });
+    const actor = await this.createActor(sessionIdentity);
+    const res = await actor.openid_create_delegation(
+      jwt,
+      salt,
+      new Uint8Array(sessionIdentity.getPublicKey().toDer()),
+      nonNullish(maxTimeToLive) ? [maxTimeToLive] : []
+    );
+    if ("Err" in res) throw new CanisterError(res.Err);
+
+    const { anchor_number, delegation_response } = res.Ok;
+
+    if ("no_such_delegation" in delegation_response) throw "No such delegation";
+
+    // create delegationIdentity from signedDelegation
+    const signedDelegation = delegation_response.signed_delegation;
+
+    const transformedDelegation: SignedDelegation = {
+      delegation: new Delegation(
+        Uint8Array.from(signedDelegation.delegation.pubkey),
+        BigInt(signedDelegation.delegation.expiration),
+        undefined
+      ),
+      signature: Uint8Array.from(
+        signedDelegation.signature
+      ) as unknown as Signature,
+    };
+
+    const chain = DelegationChain.fromDelegations(
+      [transformedDelegation],
+      sessionIdentity.getPublicKey().toDer()
+    );
+
+    const JwtSignedIdentity = DelegationIdentity.fromDelegation(
```

Comment:
It's a variable instance not a class reference)
```suggestion
    const jwtSignedIdentity = DelegationIdentity.fromDelegation(
```